frontend/src/component/events/EventDiff/EventDiff.tsx

```diff
@@ -17,7 +16,7 @@ interface IEventDiffResult {
 }
 
 interface IEventDiffProps {
-    entry: Partial<IEvent>;
+    entry: { data: unknown; preData: unknown };
```

Comment:
This one is kinda wonky. We only use `data` and `preData`. However, they're typed as `unknown` at the call site. In `IEvent`, both of these were typed as `any`, but in `EventSchema`, they're typed as `{[key: string] any} | null`.

It's almost the same, but not quite, so TS throws an error if we just sub in `EventSchema` for `IEvent` here.

So what I've done is to specify the fields we actually care about and typed them as `unknown`. We don't know anything more about them than we do about `any` properties, but at least now TS will stop you if you try to access any properties on those objects 🤷🏼